src/foam/nanos/auth/ProfilePictureView.js

```diff
@@ -239,19 +239,24 @@ foam.CLASS({
       this.addFile(file);
     },
 
-    function addFile(file) {
+    async function addFile(file) {
       if ( file.size > ( 2 * 1024 * 1024 ) ) {
```

Comment:
Flag this for configuration - future, and also multipart file upload.  We would like to handle liquid file ingestion, which are huge files.  Ask Michal how he's uploading those.

Reply:
Flag this for configuration - future. How can I flag it?